### PR TITLE
Add Marching Orders GBA and Coin Toss DS, segment thank you, and add cowbell to Coin Toss

### PR DESCRIPTION
**Pull request by GuardedLolz**
### Prerequisites
* [X] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [X] This pull request follows the code style of the project
* [X] I have tested this feature thoroughly WITH THE USE of the load-all-sounds starter
  * [Link to a video of proof of testing]()
* [X] This pull request is merging into **`dev`**

**Issues Fixed:** #146

### CODE REVIEW

**Comment by Miracle22s** [comment]
Um, move Sick Beats Megamix was added, you know.

**Comment by GuardedLolz** [comment]
I'm just moving to the side games where it belongs.

**Comment by Miracle22s** [comment]
Endless games, perhaps?

File: core/assets/sounds/cues/sickBeatsMegamix/data.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "gameID": "sickBeatsMegamix",
   "gameName": "Sick Beats (Megamix)",
-  "series": "megamix",
+  "series": "side",
```
Comment (chrislo27) [inline core/assets/sounds/cues/sickBeatsMegamix/data.json]: Side games are only for games which are exclusive. Please move this back to megamix.,

**Review by chrislo27 — CHANGES_REQUESTED** [review]
Different icon required for Marching Orders (GBA) and it has to be obvious. Move Sick Beats (Megamix) back to `megamix`.

Your lack of proof shows you don't care about testing this, seeing as you have delivered broken pull requests before. Even a screenshot would do.

**Pull request merged**